backend/tests/utils.py

```
@@ -626,30 +627,169 @@ def create_mock_history_service_dependencies():
+            
+        Returns:
+            List[SessionOverview]: List of configured SessionOverview objects
+            
+        Example:
+            overviews = MockFactory.create_mock_session_overviews(count=5)
+        """
+        from tarsy.models.converters import alert_session_to_session_overview
+        
+        # Use existing SessionFactory to create AlertSession objects, then convert
+        alert_sessions = [SessionFactory.create_test_session() for _ in range(count)]
+        return [alert_session_to_session_overview(session) for session in alert_sessions]
```

Ruff (0.12.2)

Line too long (89 > 88)

(E501)